pkg/tests/BUILD

```diff
@@ -600,6 +601,20 @@ pkg_tar(
     extension = ".bz2",
 )
 
+fake_artifact(
+    name = "a_program",
+    files = ["//tests:testdata/executable.sh"],
+    runfiles = [":BUILD"],
```

Comment:
Preceding colon is not necessary because "BUILD" is a source file, not a generated one.

Reply:
done